app/src/main/java/org/wikipedia/page/leadimages/LeadImagesHandler.java

```diff
@@ -354,4 +358,8 @@ public void dispose() {
         return callToActionIsTranslation ? callToActionTargetSummary.getPageTitle().getWikiSite().languageCode()
                 : callToActionSourceSummary.getPageTitle().getWikiSite().languageCode();
     }
+
+    public void showLeadImageInGallery() {
```

Comment:
Not sure why to create a method of doing the exact same thing of `openImageInGallery()`. Would it be better if just changes the `openImageInGallery()` to `public`.